geode-core/src/distributedTest/java/org/apache/geode/cache/ConnectionPoolDUnitTest.java

```diff
@@ -4219,360 +2990,254 @@ public void run2() throws CacheException {
   public void test028DynamicRegionCreation() {
     final String name = this.getName();
 
-    final VM client1 = VM.getVM(0);
-    final VM srv1 = VM.getVM(2);
-    final VM srv2 = VM.getVM(3);
+    final VM client1 = vm0;
+    final VM srv1 = vm2;
+    final VM srv2 = vm3;
+
+    final String key1 = name + "-key1";
+    final String value1 = name + "-val1";
+    final String key2 = name + "-key2";
+    final String value2 = name + "-val2";
+    final String key3 = name + "-key3";
+    final String value3 = name + "-val3";
+
+    // setup servers
+    Stream.of(srv1, srv2).forEach(vm -> vm.invoke("Create Cache Server", () -> {
+      createDynamicRegionCache(name, null); // Creates a new DS and Cache
+      assertThat(DynamicRegionFactory.get().isOpen()).isTrue();
+      try {
+        startBridgeServer(0);
+      } catch (IOException ugh) {
```

Comment:
Delete any catch-blocks like this and just add `throws Exception` to the test method.